components/Starknet/modules/architecture-and-concepts/pages/smart-contracts/serialization-of-cairo-types.adoc

```diff
@@ -162,8 +164,8 @@ Now consider instantiations of the `Week` enum's variants as shown in the table
 |===
 |Instance |Description |Serialization
 
-|`Week::Sunday` | Index=`0`. The variant's type is the unit type. | `[1]`
-|`Week::Monday(5)` a| Index=`1`. The variant's type is `u256`.| `[2,5,0]`
+|`Week::Sunday` | Index=`0`. The variant's type is the unit type. | `[0]`
+|`Week::Monday(5)` a| Index=`1`. The variant's type is `u256`.| `[1,5,0]`
```

Comment:
It is not immediately intuitive that the `5, 0` are a result of uint256 low and high.
Maybe underscore it somehow
```
[ 1, 5, 0]
  |  |  |
  |  |  unit256 high
  |  uint256 low
  variant index
```